components/crud-web-apps/jupyter/backend/apps/common/yaml/spawner_ui_config.yaml

```diff
@@ -25,14 +25,14 @@ spawnerFormDefaults:
       - public.ecr.aws/j1r0q0g6/notebooks/notebook-servers/jupyter-pytorch-cuda-full:master-1831e436
       - public.ecr.aws/j1r0q0g6/notebooks/notebook-servers/jupyter-tensorflow-full:master-1831e436
       - public.ecr.aws/j1r0q0g6/notebooks/notebook-servers/jupyter-tensorflow-cuda-full:master-1831e436
-  imageVSCode:
-    # The container Image for the user's VS-Code Server
+  imageGroupOne:
+    # The container Image for the user's Group One Server
     value: public.ecr.aws/j1r0q0g6/notebooks/notebook-servers/codeserver-python:master-1831e436
     # The list of available standard container Images
     options:
       - public.ecr.aws/j1r0q0g6/notebooks/notebook-servers/codeserver-python:master-1831e436
-  imageRStudio:
-    # The container Image for the user's RStudio Server
+  imageGroupTwo:
+    # The container Image for the user's Group Two Server
```

Comment:
Can you add a comment here that for images in this group the backend will be:
1. Adding the `notebooks.kubeflow.org/http-headers-request-set` annotation to the CR, for setting the `X-RStudio-Root-Path: ${ISTIO_PREFIX}`  header to each request to the running container
2. Adding the `notebooks.kubeflow.org/http-rewrite-uri` annotation to the CR, to rewrite the path that ends up in the running container to be `/`

This will help exposing to the users the constraints and logic that will be applied to images that belong to that group

Reply:
How about:
```yaml
# The annotation `notebooks.kubeflow.org/http-rewrite-uri: /`
# is applied to notebook in this group, configuring
# the Istio rewrite for containers that host their web UI at `/`
# The annotation `notebooks.kubeflow.org/http-headers-request-set`
# is applied to notebook in this group, configuring Istio
# to add the `X-RStudio-Root-Path` header to requests
```

Reply:
LGTM